.changeset/deprecate-dropdownmenu.md

```diff
@@ -0,0 +1,107 @@
+---
+'@primer/react': major
+---
+
+### DropdownMenu
+
+DropdownMenu has been deprecated in favor of ActionMenu with ActionList
```

Comment:
Thinking out loud. I don't think we need to advertise ActionList in this way as that's implementation detail.. or is this one of those `Built with ActionList™ 💪` kind of things 😄